index.html

```diff
@@ -1255,6 +1239,231 @@ <h3>
+                "ar":    {"value": "منتقي الألوان", "dir": "rtl"}
+              }
+            }
+          </pre>
+        </aside>
+        <p>
+          [=manifest/*_localized=] members contain a [=language map=] that
+          defines [=localized values=] for the given [=localizable member=] in
+          the application. The user agent SHOULD use the user's localization
+          settings to select the [=localized value=] whose language tag best
+          matches the user's preference. When no such [=localized value=] is
+          available, the default representation is used.
+        </p>
+        <section>
+          <h3>
+            Localizing text resources
+          </h3>
+          <p>
+            For [=manifest/*_localized=] members that accept a [=string=] as
+            their value, the [=localized value=] can either be a plain
+            localized [=string=] or an [=ordered map=] containing the following
+            members:
+          </p>
+          <dl>
+            <dt>
+              `value`
+            </dt>
+            <dd>
+              The localized [=string=].
+            </dd>
+            <dt>
+              `lang` (optional)
+            </dt>
+            <dd>
+              A [=language tag=].
+            </dd>
+            <dt>
+              `dir` (optional)
+            </dt>
+            <dd>
+              The [=text-direction=].
+            </dd>
+          </dl>
+          <p>
+            When a plain [=string=] is used, or when the `lang` or `dir`
+            members are omitted, `lang` falls back the [=language tag=] of the
+            [=language map=] key, and `dir` falls back to the [=manifest/dir=]
+            member of the [=manifest=].
+          </p>
+          <aside class="note">
+            <p>
+              The `dir` member must be specified if the direction of a
+              localized string differs from the default direction set in the
+              manifest ([=manifest/dir=] member). Right-to-left text will
+              require specific direction settings if the manifest's default
+              direction is left-to-right, and vice versa.
+            </p>
+            <p>
+              To support multilingual content and ensure proper display and
+              accessibility, it is possible to specify a different [=language
+              tag=] for a [=localized value=]. This is needed for situations
+              where a term or text must be presented in a language different
+              from the user's set language. For example, this helps ensure that
+              an application name is correctly pronounced by assistive
+              technology, even if it is in a foreign language.
+            </p>
+          </aside>
+          <p>
+            To <dfn>process a `*_localized` text member</dfn>, given [=ordered
+            map=] |json:ordered map|, [=ordered map=] |map:ordered map|,
```

Comment:
optional:

```suggestion
            map=] |json:ordered map|, [=ordered map=] |outputMap:ordered map|,
```
(and change references below)

This helps readability for me, knowing this argument is populated as the output of this algo

Reply:
I tried to be consistent with the other algos in the spec that use “map” currently. If we change this, should we change all uses of “map”?

Reply:
I'm not sure - I don't know if it's so important to keep argument names consistent, especially for this patch. I think just update these ones.

Reply:
oh, good point... I'm kinda regretting the way I ended specifying passing in the output map :( That's a very "gecko" thing to do, and I should really switch all these back to retiring a parser map... anyway, let's do "map" for now, and then we can fix all of them later.